flatliners/comparisonscore.py

```diff
@@ -0,0 +1,46 @@
+from .baseflatliner import BaseFlatliner
+
+
+class ComparisonScore(BaseFlatliner):
+    def __init__(self):
+        super().__init__()
+
+        self.score = dict()
+        self.clusters = dict()
+        self.versions = dict()
+
+
+    def on_next(self, x):
+        """ update calculate std dev for cluster
+        """
+
+        # determine if version metric or if it is a cluster metric
+        version_record = list(x.keys())[0] == 'version'
+        cluster_record = list(x.keys())[0] == 'cluster'
+        # Determine the version and resource name for the current record
+        version_id = x['version']
+        resource = x['resource']
+
+        # for version records, collect the average standard deviation value for
+        # each resource
+        if version_record:
```

Comment:
these are great for a function. and name the function properly to hint at what the function is doing. so `process_version_record` is too generic :)

Reply:
@durandom, I made the changes you requested. Let me know if there is anything else that needs to be updated here.